src/App.js

```diff
@@ -23,23 +24,46 @@ function App() {
     <WalletProvider>
       <BrowserRouter>
         <ActivityProvider>
-        <Main layout={false} theme={theme}>
-          <MainView>
-            <Redirect from='/' to='/dashboard' />
-            <Switch>
-              <Route exact path='/profile' component={Dashboard} />
-              <Route exact path='/leaderboard' component={Tasks} />
-              <Route exact path='/dashboard' component={Disputes} />
-              <Route exact path='/sample' component={Sample} />
-
-              <Route component={ErrorPage} />
-            </Switch>
-          </MainView>
-        </Main>
+          <Switch>
+            <Route exact path='/login' component={LoginPage} />
```

Comment:
Point the root `/` path also to LoginPage for now

Reply:
Done